Playmate/View Controllers/Home Tab/HomeViewController.m

```diff
@@ -187,11 +195,18 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
          VC.sessionDetails = data;
      }
      
-     if ([sender isMemberOfClass:[UIButton class]]) {
+     if ([segue.identifier isEqualToString:@"toCalendar"]) {
          CalendarViewController *VC = [segue destinationViewController];
          VC.rawSessionList = self.sessionList;
      }
      
  }
 
+- (IBAction)goToProfile:(id)sender {
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    UITabBarController *tabBarController = [storyboard instantiateViewControllerWithIdentifier:@"TabBarController"];
```

Comment:
I feel confused of us instantiate a tabBar, and then using it. 
In my mind if you modify a new tabBar, it wont work. 
I have get access to the tabBar by calling:
`self. tabBarController ` 
or
`self.neavigationController.rootViewController.tabBarController` (or something similar)